src/actions/controller.ts

```diff
@@ -23,17 +27,44 @@ export class ActionsController {
       }
     }
 
+    public static async performActionsFromParams() {
+      const logger = getLogger().getChildLogger({label: ActionsController.loggerLabel});
```

Comment:
this logger is used twice in this file, make logger member instead of loggerLabel